File: debs/CPU/CCModule/CPUthermalFreqCCModule.m
#import "CPUthermalFreqCCModule.h"
#import "CPUthermalFreqCCModuleViewController.h"

@implementation CPUthermalFreqCCModule

@synthesize contentViewController = _contentViewController;

- (UIViewController<CCUIContentModuleContentViewController> *)contentViewController {
    if (!_contentViewController) {
        _contentViewController = [[CPUthermalFreqCCModuleViewController alloc] init];
    }
    return _contentViewController;
}

- (UIViewController *)backgroundViewController {
    return nil;
}

@end

File: debs/CPU/CCModule/CPUthermalFreqCCModule.swift
import Foundation
import UIKit

@objc(CPUthermalFreqCCModule)
public final class CPUthermalFreqCCModule: NSObject, CCUIContentModule {
    private lazy var moduleContentViewController: UIViewController & CCUIContentModuleContentViewController = CPUthermalFreqCCModuleViewController()

    public var contentViewController: UIViewController & CCUIContentModuleContentViewController {
        moduleContentViewController
    }

}
